concepts-wip/bools/.meta/config.json

```
@@ -0,0 +1,5 @@
+{
+  "blurb": "Crystal has a Bool type to represent true and false and a bool can only be true or false. The bool type support AND, OR, and NOT operators",
```

In many languages, whether a type's name is uppercase or lowercase has a semantic difference. I would suggest to choose a consistent casing and be pedantic about always using that casing. Even if it doesn't matter in Crystal, it will make it easier on learners.

I think the official documentation uses uppercase for types, so:
```suggestion
  "blurb": "Crystal has a Bool type to represent true and false and a Bool can only be true or false. The Bool type support AND, OR, and NOT operators",
```

(This feedback probably applies to more than just this file)

---

Yes, you are right. All types in crystal is either a module, class or struct (and sometimes a combination). The naming scheme for those are PascalCased.